rfc/20250728-opentofu-providers/05-registry-integration.md

```diff
@@ -0,0 +1,151 @@
+# Registry Integration
+
+## Summary
+
+This document defines a simplified registry integration model for the new proposed OpenTofu providers focused on **discovery rather than distribution**. The registry serves as a lightweight catalog that helps users find providers and learn how to use them, while leaving versioning, installation, and distribution to external systems (package managers, version control, etc.).
```

Comment:
I hesitate to call this a registry.  I think the term would just confuse users, it certainly confused me.

Maybe a search index? A catalogue?  Something that's not registry.